src/components/OrderDetailWidget/OrderDetailWidget.tsx

```diff
@@ -120,6 +121,7 @@ const OrderDetailWidget: FC<OrderDetailWidgetProps> = ({ order }) => {
 
   const handleCopyButtonClick = () => {
     navigator.clipboard.writeText(window.location.toString());
+    notifyCopySuccess();
   };
```

Comment:
Rename `writeAddressToClipboard` to `writeTextToClipboard` and check if write to clipboard is actually success. Now you're just assuming it's success.